spec/system/search_signs_spec.rb

```diff
@@ -48,10 +48,6 @@
   def submit_search(term)
     visit root_path
     form = find(".hero-unit form.search-bar")
-    form.fill_in "Search signs", with: term
-
-    # Artificially submit the form. Normally the browser does this, but
-    # Rack::Test can't.
-    Capybara::RackTest::Form.new(page.driver, form.native).submit({})
+    form.fill_in "Search signs", with: "#{term}\n"
```

Comment:
note: the previous implementation was erroring about param args mismatch, but [now Capybara will automatically submit single-input forms](https://github.com/teamcapybara/capybara/commit/6b05e5cb30e9807483c2641e9a60d89692bb0487) if the value ends with a newline